backend/src/main/resources/templates/Rhosak/actionRequiredBody.html

```diff
@@ -1,6 +1,48 @@
 {#include Rhosak/insightsEmailBody}
-{#content-title} Action Required - {action.context.system_check_in.toUtcFormat()}{/content-title}
+{#content-title}Action Required - {action.context.system_check_in.toUtcFormat()}{/content-title}
 {#content-body}
-<!-- TODO put email content here -->
+<tr>
+    <td class="rh-content__block">
+        <p>This notification is for your {action.events.size()} Kafka {#if action.events.size() is 1} instance {#else} instances{/if} below.</p>
```

Comment:
What is the structure of the `action` object here? What does events.size() represent and where does it originate from?

Reply:
The action object looks like below:
```
{
  "bundle": "application-services",
  "application": "rhosak",
  "event_type": "action-required",
  "timestamp": "2021-08-05T16:21:14.243",
  "account_id": "5758117",
  "context": "{\"issue_description\":\"description of issue goes here\",\"system_check_in\":\"2021-08-05T16:21:12.953036\",\"action_required\":\"the required action goes here\"}",
   "events": [
    {
      "metadata": {},
      "payload": "{\"id\":\"3df53241-3e09-481b-a322-4892caaaaadc\",\"name\":\"kafka-instance-name-1\"}"
    },
    {
      "metadata": {},
      "payload": "{\"id\":\"6c5e8451-a40a-4bb7-ab9a-0cb10a4c577d\",\"name\":\"kafka-instance-name-2\"}"
    },
    {
      "metadata": {},
      "payload": "{\"id\":\"b4c6378a-c1fb-4d3e-8e9b-7e5bdfc09dd3\",\"name\":\"kafka-instance-name-3\"}"
    }
  ]
}
```

It is also added in the JIRA. The events is part of the payload sent as far as I know @gwenneg @josejulio would you confirm?

Reply:
It's correct, that's the format of the data sent to trigger a notification.

Reply:
@machi1990 Just to clarify what you mean by `The events is part of the payload sent`. Do you mean the events are supplied with the request body sent to the notification API?

Reply:
Yes, that is correct @davidffrench 

See the example https://core-platform-apps.pages.redhat.com/notifications-docs/dev/apps-guide/send-notification.html#_ci_no_authentication on the docs.

Reply:
Thanks @machi1990